src/completer/widget.ts

```diff
@@ -436,15 +436,18 @@ class CompleterWidget extends Widget {
       return;
     }
 
+    const editor = this._editor;
     const { coords, charWidth, lineHeight } = model.original;
+    const position = editor.getPositionAt(model.cursor.start);
+    const anchor = editor.getCoordinateForPosition(position) as ClientRect;
```

Comment:
I would suggest doing that, but I wanted to discuss first and confirm it's a good idea.